sparkmagic/sparkmagic/tests/test_reliablehttpclient.py

```diff
@@ -2,18 +2,21 @@
 # Distributed under the terms of the Modified BSD License.
 
 from mock import patch, PropertyMock, MagicMock
-from nose.tools import raises, assert_equals, with_setup
+from nose.tools import raises, assert_equals, with_setup, assert_is_not_none, assert_false, assert_true
 import requests
+from pandas.util.testing import assertIsInstance
```

Comment:
You are still using `assertIsInstance`. Previous comment:

>Please use `assert isinstance()` instead: https://docs.python.org/2/library/functions.html#isinstance since you are not checking Pandas dataframes here.